help/include/automatic-billing.md

```diff
@@ -7,14 +7,33 @@ With automatic billing, you automatically purchase a Zulip license for each user
 in your organization at the start of each billing period (month or year), and
 whenever you need additional licenses.
 
-- [Deactivating a user](/help/deactivate-or-reactivate-a-user) frees up their
-  license for reuse. No refunds are given for unused licenses you have
-  purchased. For example, if you start a billing period with 10 users,
-  deactivate 5 users, and then add 2 new users, you will be billed for 10
-  licenses for the entire billing period.
+- [Deactivating a user](/help/deactivate-or-reactivate-a-user) frees
+  up their license for reuse. No refunds are given for unused licenses
+  you have purchased.
 
-- If you have no free licenses when you **add a user**, you will be automatically
-  billed for an additional license. You will be charged only for the remaining
-  part of the billing period. For example, if you are on a $8/user/month monthly
-  plan, you will be billed $4 for a user added half way through the month. Each
-  month, you will see a single charge for all licenses added that month.
+- If you have no free licenses when you **add a user**, you will be
+  automatically billed for an additional license. You will be charged
+  a pro-rated rate only for the remaining part of the billing
+  period. Each month, you will see a single invoice covering all
+  licenses added that month.
+
+For example, suppose you start a billing month on a monthly plan with
+15 users on Zulip Basic at $3.50/month. You'll be billed at the start
+of the month for 15 licenses ($52.50). Suppose during that billing
+month, you:
+
+* Add 1 user five days before the end of the billing month. This requires
+  one additional license, for a total of 16.
+* Deactivate 4 users the next day, leaving you with 12 users and 16
+  licenses.
+* Add 6 more users on the second-to-last day of the billing month,
+  bringing you to 18 users, and requiring 2 additional licenses for a
+  total of 18.
+* Deactivate 2 users on the last day of the billing month, leaving you
+  with 16 users and 18 licenses.
+
```

Comment:
This example, while comprehensive, I feel may be unnecessarily complicated for someone just trying to get an initial basic idea. Perhaps it's better to structure this to show two examples, first one very basic and then second one comprehensive? E.g.

For example, suppose you start a billing month on a monthly plan with
15 users on Zulip Basic at $3.50/month. You'll be billed at the start
of the month for 15 licenses ($52.50). Suppose during that billing
month, you:

* Add 1 user five days before the end of the billing month. This requires
  one additional license, for a total of 16.

Then, at the end of the billing month, you'll receive a single invoice
for 16 licenses at $3.50/month ($56, aka the initial purchase for the
next month) as well as the previous billing month's pro-rated
purchase of 1 license for 5 days.

Next, consider a more complicated scenario, involving a mix of deactivating some users as well as adding new ones:

...